2023-02-13 20:31:53 | ERROR    | Error on main market performance, invalid index to scalar variable. | main | market_performers.py:134
2023-02-13 20:32:40 | ERROR    | Error on main market performance, invalid index to scalar variable. | main | market_performers.py:135
2023-02-13 21:28:45 | ERROR    | Error on main market performance, unsupported operand type(s) for -: 'str' and 'str' | main | market_performers.py:135
2023-02-13 21:33:25 | ERROR    | Error on main market performance, unsupported operand type(s) for /: 'str' and 'str' | main | market_performers.py:135
2023-02-15 20:16:38 | ERROR    | Error on main market performance, MomentumRank._performance_calculations() got an unexpected keyword argument 'min_vol_USD' | main | market_performers.py:71
2023-02-15 20:19:57 | ERROR    | Error on main market performance, 'BTC'       | main | market_performers.py:73
2023-02-15 20:20:05 | ERROR    | Error on main market performance, 'BTC/USDT'  | main | market_performers.py:73
2023-02-15 20:20:28 | ERROR    | Error on main market performance, 'BTC/USDT'  | main | market_performers.py:74
2023-02-15 20:21:06 | ERROR    | Error on main market performance, 'BTC/USDT'  | main | market_performers.py:74
2023-02-15 21:08:50 | ERROR    | Error on main market performance, PerformanceCalculation.performance_calculations() missing 3 required positional arguments: 'days_windows', 'min_vol_usd', and 'min_vol_btc' | main | market_performers.py:70
2023-02-15 21:35:31 | ERROR    | Error on main market performance, -1          | main | market_performers.py:72
2023-02-17 20:01:05 | ERROR    | Error on main market performance, 'avg_vol_fast' | main | market_performers.py:72
2023-02-17 20:28:33 | ERROR    | Error on main full history, object of type 'int' has no len() | main | get_full_history.py:94
2023-02-17 20:28:34 | ERROR    | Error on main full history, object of type 'int' has no len() | main | get_full_history.py:94
2023-02-17 20:28:34 | ERROR    | Error on main full history, object of type 'int' has no len() | main | get_full_history.py:94
2023-02-17 20:28:34 | ERROR    | Error on main full history, object of type 'int' has no len() | main | get_full_history.py:94
2023-02-17 20:28:34 | ERROR    | Error on main full history, object of type 'int' has no len() | main | get_full_history.py:94
2023-02-17 20:33:48 | ERROR    | Error on main market performance, 'avg_vol_fast' | main | market_performers.py:73
2023-02-28 07:59:02 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 07:59:41 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 08:13:32 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 08:18:20 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:68
2023-02-28 08:26:42 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 08:26:54 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 08:28:21 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 08:29:15 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 08:29:24 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'pairs_list' | main | market_performers.py:69
2023-02-28 08:38:02 | ERROR    | Error on main market performance, 'PerformanceRankAnalysis' object has no attribute 'QUANTILE_DROP' | main | market_performers.py:69
2023-03-01 08:59:56 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:57 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:57 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:57 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:57 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:57 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:57 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:58 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 08:59:59 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:00 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:01 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:02 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:03 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:04 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:05 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:06 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:07 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:08 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:09 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:10 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:11 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:12 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:13 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:14 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:15 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:16 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:17 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:18 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:19 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:20 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:21 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:22 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:23 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:29 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:30 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-01 09:00:30 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657630273} | main | get_full_history.py:98
2023-03-01 09:00:30 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657630537} | main | get_full_history.py:98
2023-03-01 09:00:30 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657630559} | main | get_full_history.py:98
2023-03-01 09:00:30 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657630730} | main | get_full_history.py:98
2023-03-01 09:00:30 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657630922} | main | get_full_history.py:98
2023-03-01 09:00:31 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657631111} | main | get_full_history.py:98
2023-03-01 09:00:31 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657631118} | main | get_full_history.py:98
2023-03-01 09:00:31 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657631301} | main | get_full_history.py:98
2023-03-01 09:00:31 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657631331} | main | get_full_history.py:98
2023-03-01 09:00:31 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657631493} | main | get_full_history.py:98
2023-03-01 09:00:31 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657631522} | main | get_full_history.py:98
2023-03-01 09:00:31 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657631689} | main | get_full_history.py:98
2023-03-01 09:00:32 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657632082} | main | get_full_history.py:98
2023-03-01 09:00:32 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657632229} | main | get_full_history.py:98
2023-03-01 09:00:32 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657632630} | main | get_full_history.py:98
2023-03-01 09:00:32 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657632774} | main | get_full_history.py:98
2023-03-01 09:00:33 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657633327} | main | get_full_history.py:98
2023-03-01 09:00:33 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657633358} | main | get_full_history.py:98
2023-03-01 09:00:33 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657633516} | main | get_full_history.py:98
2023-03-01 09:00:33 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657633713} | main | get_full_history.py:98
2023-03-01 09:00:34 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657634090} | main | get_full_history.py:98
2023-03-01 09:00:34 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657634258} | main | get_full_history.py:98
2023-03-01 09:00:34 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657634803} | main | get_full_history.py:98
2023-03-01 09:00:34 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657634811} | main | get_full_history.py:98
2023-03-01 09:00:34 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657634996} | main | get_full_history.py:98
2023-03-01 09:00:35 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657635537} | main | get_full_history.py:98
2023-03-01 09:00:35 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657635545} | main | get_full_history.py:98
2023-03-01 09:00:35 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657635725} | main | get_full_history.py:98
2023-03-01 09:00:35 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657635754} | main | get_full_history.py:98
2023-03-01 09:00:35 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657635913} | main | get_full_history.py:98
2023-03-01 09:00:36 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657636107} | main | get_full_history.py:98
2023-03-01 09:00:36 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657636468} | main | get_full_history.py:98
2023-03-01 09:00:36 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657636655} | main | get_full_history.py:98
2023-03-01 09:00:36 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657636658} | main | get_full_history.py:98
2023-03-01 09:00:36 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657636847} | main | get_full_history.py:98
2023-03-01 09:00:36 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657636878} | main | get_full_history.py:98
2023-03-01 09:00:37 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657637388} | main | get_full_history.py:98
2023-03-01 09:00:37 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657637576} | main | get_full_history.py:98
2023-03-01 09:00:37 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657637599} | main | get_full_history.py:98
2023-03-01 09:00:37 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657637767} | main | get_full_history.py:98
2023-03-01 09:00:37 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657637786} | main | get_full_history.py:98
2023-03-01 09:00:37 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657637961} | main | get_full_history.py:98
2023-03-01 09:00:37 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657637986} | main | get_full_history.py:98
2023-03-01 09:00:38 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657638185} | main | get_full_history.py:98
2023-03-01 09:00:38 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657638502} | main | get_full_history.py:98
2023-03-01 09:00:38 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657638694} | main | get_full_history.py:98
2023-03-01 09:00:38 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657638884} | main | get_full_history.py:98
2023-03-01 09:00:38 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657638907} | main | get_full_history.py:98
2023-03-01 09:00:39 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657639428} | main | get_full_history.py:98
2023-03-01 09:00:39 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657639446} | main | get_full_history.py:98
2023-03-01 09:00:39 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657639619} | main | get_full_history.py:98
2023-03-01 09:00:39 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657639810} | main | get_full_history.py:98
2023-03-01 09:00:39 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657639999} | main | get_full_history.py:98
2023-03-01 09:00:39 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657639999} | main | get_full_history.py:98
2023-03-01 09:00:40 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657640218} | main | get_full_history.py:98
2023-03-01 09:00:40 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657640408} | main | get_full_history.py:98
2023-03-01 09:00:40 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657640544} | main | get_full_history.py:98
2023-03-01 09:00:40 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657640598} | main | get_full_history.py:98
2023-03-01 09:00:40 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657640732} | main | get_full_history.py:98
2023-03-01 09:00:40 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657640795} | main | get_full_history.py:98
2023-03-01 09:00:40 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657640930} | main | get_full_history.py:98
2023-03-01 09:00:41 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657641484} | main | get_full_history.py:98
2023-03-01 09:00:41 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657641506} | main | get_full_history.py:98
2023-03-01 09:00:41 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657641677} | main | get_full_history.py:98
2023-03-01 09:00:41 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657641703} | main | get_full_history.py:98
2023-03-01 09:00:41 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657641877} | main | get_full_history.py:98
2023-03-01 09:00:41 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657641911} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657642070} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657642101} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657642261} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657642291} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657642450} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657642642} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657642833} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643000} | main | get_full_history.py:98
2023-03-01 09:00:42 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643027} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643188} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643223} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643386} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643415} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643572} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643605} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643762} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643799} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643954} | main | get_full_history.py:98
2023-03-01 09:00:43 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657643989} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644145} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644351} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644530} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644541} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644721} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644740} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644910} | main | get_full_history.py:98
2023-03-01 09:00:44 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657644943} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645103} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645293} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645483} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645490} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645675} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645702} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645864} | main | get_full_history.py:98
2023-03-01 09:00:45 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657645896} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646084} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646273} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646411} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646463} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646602} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646653} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646792} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646840} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657646983} | main | get_full_history.py:98
2023-03-01 09:00:46 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647029} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647174} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647219} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647364} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647411} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647567} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647600} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647757} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647788} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647955} | main | get_full_history.py:98
2023-03-01 09:00:47 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657647975} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648173} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648366} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648486} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648560} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648676} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648745} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648869} | main | get_full_history.py:98
2023-03-01 09:00:48 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657648936} | main | get_full_history.py:98
2023-03-01 09:00:49 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657649059} | main | get_full_history.py:98
2023-03-01 09:00:49 | ERROR    | Error on main full history, bybit {"retCode":10001,"retMsg":"Time params invalid","result":{},"retExtInfo":{},"time":1677657649126} | main | get_full_history.py:98
2023-03-08 21:38:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:24 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:25 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:26 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:27 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-03-08 21:38:28 | ERROR    | Error on main full history, list index out of range | main | get_full_history.py:98
2023-04-01 00:00:45 | ERROR    | Error on main full history, ThreadPoolEngine doesn't expect arguments ['max_workers'] | _get_vbt_desired_history | get_history.py:98
2023-04-01 00:02:57 | ERROR    | Error on main full history, 'int' object is not subscriptable | _get_vbt_desired_history | get_history.py:98
2023-04-01 00:03:01 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-01 00:10:03 | ERROR    | Error on main full history, CCXTData.fetch_symbol() got an unexpected keyword argument 'init_kwargs' | _get_vbt_desired_history | get_history.py:99
2023-04-02 00:20:50 | ERROR    | Error on main full history, unsupported operand type(s) for +: 'NoneType' and 'datetime.timedelta' | get_full_history | get_history.py:58
2023-04-02 00:20:53 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-02 00:21:33 | ERROR    | Error on main full history, local variable 'delegate_data_storing' referenced before assignment | get_full_history | get_history.py:58
2023-04-02 00:21:38 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-02 00:23:44 | ERROR    | Error on main full history, can't compare offset-naive and offset-aware datetimes | get_full_history | get_history.py:58
2023-04-02 00:23:47 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-02 20:59:26 | ERROR    | Error on main full history, can't compare offset-naive and offset-aware datetimes | get_full_history | get_history.py:58
2023-04-02 20:59:29 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-02 21:02:50 | ERROR    | Error on main full history, can't compare offset-naive and offset-aware datetimes | get_full_history | get_history.py:58
2023-04-02 21:02:54 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-02 21:05:29 | ERROR    | Error on main full history, can't compare offset-naive and offset-aware datetimes | get_full_history | get_history.py:58
2023-04-02 21:05:33 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-02 21:30:44 | ERROR    | Error on main market performance, 'Series' object has no attribute 'pair' | main | market_performers.py:66
2023-04-02 23:04:38 | ERROR    | Error on main market performance, 'tuple' object has no attribute 'popitem' | main | market_performers.py:66
2023-04-02 23:05:17 | ERROR    | Error on main market performance, 'tuple' object has no attribute 'popitem' | main | market_performers.py:66
2023-04-02 23:08:13 | ERROR    | Error on main market performance, 'tuple' object has no attribute 'popitem' | main | market_performers.py:66
2023-04-02 23:08:51 | ERROR    | Error on main market performance, 'str' object has no attribute 'popitem' | main | market_performers.py:66
2023-04-03 08:22:20 | ERROR    | Error on main market performance, _PerformanceCalculation.performance_calculations() missing 1 required positional argument: 'coin_history_df' | main | market_performers.py:66
2023-04-03 08:28:55 | ERROR    | Error on main market performance, 'close'     | main | market_performers.py:66
2023-04-03 09:00:08 | ERROR    | Error on main market performance, 'pairs_list' | main | market_performers.py:66
2023-04-03 18:49:35 | ERROR    | Error on main full history, No symbols could be fetched | get_full_history | get_history.py:63
2023-04-03 18:49:38 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-03 19:02:21 | ERROR    | Error on main full history, No symbols could be fetched | get_full_history | get_history.py:63
2023-04-03 19:02:23 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-03 19:02:50 | ERROR    | Error on main full history, No symbols could be fetched | get_full_history | get_history.py:63
2023-04-03 19:02:53 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-03 19:03:11 | ERROR    | Error on main full history, No symbols could be fetched | get_full_history | get_history.py:63
2023-04-03 19:03:13 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-03 19:03:33 | ERROR    | Error on main full history, No symbols could be fetched | get_full_history | get_history.py:63
2023-04-03 19:03:35 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-03 19:07:03 | ERROR    | Error on main full history, No symbols could be fetched | get_full_history | get_history.py:63
2023-04-03 19:07:06 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-03 19:08:18 | ERROR    | Error on main full history, No symbols could be fetched | get_full_history | get_history.py:63
2023-04-03 19:08:20 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-03 19:11:49 | ERROR    | Error on main full history, division by zero  | get_full_history | get_history.py:63
2023-04-03 19:11:51 | ERROR    | Error on main market performance, 'NoneType' object has no attribute 'data' | main | market_performers.py:66
2023-04-23 23:16:45 | ERROR    | Error on main market performance, name '_PerformanceCalculation' is not defined | main | market_performers.py:72
2023-04-26 22:02:11 | ERROR    | Error on main full history, list index out of range | get_full_history | get_history.py:73
2023-04-26 22:02:57 | ERROR    | Error on main full history, list index out of range | get_full_history | get_history.py:73
2023-04-26 22:08:11 | ERROR    | Error on main full history, list index out of range | get_full_history | get_history.py:73
